updatePassword.php

```diff
@@ -0,0 +1,19 @@
+<?php
+session_start();
+$password = $_POST["password"];
+$password = hash('sha256',$password);
+if (isset($_SESSION["forgotPasswordEmail"])) {
+  $email = $_SESSION["forgotPasswordEmail"];
```

Comment:
@aniarya82 `unset` this variable right here, and not inside the SQL block. If the SQL query doesn't come back with a valid result, then the user will start all over again.